src/math/Math.js

```diff
@@ -259,21 +259,28 @@ FORGE.Math.sphericalToCartesian = function(radius, theta, phi)
  * @param {number} x - x
  * @param {number} y - y
  * @param {number} z - z
+ * @param {string} [unit=radian] - The unit used to return spherical
```

Comment:
What unit would you like to add to the return value of sphericalToCartesian ? Cartesian result is world unit only, am I right?